doc/mgr/orchestrator.rst

```diff
@@ -116,6 +116,34 @@ Service Commands::
     ceph orch <start|stop|restart|redeploy|reconfig> <service_name>
 
 
+Options
+=======
+
+.. option:: start
+
+   Start the daemon on the corresponding host.
+
+.. option:: stop
+
+   Stop the daemon on the corresponding host.
+
+.. option:: restart
+
+   Restart the daemon on the corresponding host.
+
+.. option:: redeploy
+
+   Redeploy the daemon on the corresponding host. This will recreate the daemon directory
+   structure under ``/var/lib/ceph/<fsid>/<daemon-name>`` if it doesn't exist, refresh its
+   configuration files, regenerate its unit-files and restarts the systemd daemon.
```

Comment:
`regenerate its unit files and restart the \`\`systemd\`\` daemon.`

But doesn't this assume that one is running traditional daemons, not containers?  I'd like to see the wording cover other deployment types.

Reply:
@anthonyeleven thanks for highlighting this point. I'm not sure if this applies to old daemons as well. I'll check and update the docs accordingly.

Reply:
@anthonyeleven this doc section applies to `cephadm` only. Internally, cephadm create/use systemd daemons to run ceph containers. I think no extra clarification is needed here since when talking about cephadm we are already assuming a modern/container based ceph deployment.

Reply:
What about the `simple` scenario?

Reply:
please, can you clarify what you mean with `simple` scenario? do you refer to legacy ceph daemons?

Reply:
Yes.

Reply:
@anthonyeleven I see, in the adoption is covered by anther doc section [cephadm adopt](https://docs.ceph.com/en/quincy/cephadm/adoption/#adoption-process) so these commands `(restart|reconfig|...)` don't apply to this scenario. I'll add a note clarifying that these commands apply to new containerized cephadm daemons only.